include/bluetooth/peer_manager/peer_manager.h

```diff
@@ -773,7 +773,7 @@ uint32_t pm_peers_delete(void);
  * @retval NRF_ERROR_INVALID_STATE  If the Peer Manager is not initialized.
  * @retval NRF_ERROR_INTERNAL       If an internal error occurred.
  * @retval NRF_ERROR_NOT_SUPPORTED  If peer rank functionality has been disabled via the @ref
- *                                  PM_PEER_RANKS_ENABLED configuration option.
+ *                                  PM_PEER_RANKS configuration option.
```

Comment:
@ before will link to it

Reply:
Please have another look, this is addressed in a separate commit.